packages/react-components/react-tree/library/src/components/Tree/Tree.cy.tsx

```diff
@@ -128,6 +128,27 @@ describe('Tree', () => {
       cy.get(`#action`).realClick();
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32042&commitId=9c841ce8197d42af15b3c4cdd169a70bf13f98b0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>